official/utils/logs/hooks_helper.py

```diff
@@ -157,3 +158,10 @@ def get_logging_metric_hook(benchmark_log_dir=None,
     'examplespersecondhook': get_examples_per_second_hook,
     'loggingmetrichook': get_logging_metric_hook,
 }
+
+HOOKS_ALIAS = {
+    "lt": "LoggingTensorHook",
+    "p": "ProfilerHook",
+    "eps": "ExamplesPerSecondHook",
+    "lm": "LoggingMetricHook",
+}
```

Comment:
This is something I've wanted for a while, but the formatting in argparse forbade a good help message for it. I added an "h" to the end of them all, so ProfilerHook is "ph" instead of "p" to make it a little harder to mess up.